sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ServiceBusManagementClientImplIntegrationTests.java

```diff
@@ -0,0 +1,257 @@
+            .assertNext(response -> {
+                final QueueDescriptionResponse deserialize = deserialize(response, QueueDescriptionResponse.class);
+                assertNotNull(deserialize);
+                assertNotNull(deserialize.getContent());
+
+                final QueueDescription properties = deserialize.getContent().getQueueDescription();
+                assertNotNull(properties);
+                assertFalse(properties.getLockDuration().isZero());
+            })
+            .verifyComplete();
+    }
+
+    /**
+     * Verifies we can create a queue.
+     */
+    @Test
+    void createQueue() {
+        // Arrange
+        String queueName = testResourceNamer.randomName("test", 7);
+        QueueDescription description = new QueueDescription().setMaxDeliveryCount(15);
+        CreateQueueBody createEntity = new CreateQueueBody();
+        CreateQueueBodyContent content = new CreateQueueBodyContent()
+            .setType("application/xml")
+            .setQueueDescription(description);
+        createEntity.setContent(content);
+
+        logger.info("Creating queue: {}", queueName);
+
+        // Act & Assert
+        StepVerifier.create(queuesClient.putWithResponseAsync(queueName, createEntity, null, Context.NONE))
+            .assertNext(response -> {
+                Object body = response.getValue();
+                QueueDescription deserialize = null;
+                try {
+                    deserialize = new ServiceBusManagementSerializer()
+                        .deserialize(String.valueOf(body), QueueDescription.class);
+                } catch (IOException e) {
+                    fail("An exception was thrown. " + e);
+                }
+
+                assertNotNull(deserialize);
+            })
+            .verifyComplete();
+    }
+
+    /**
+     * Verifies we can delete a queue.
+     */
+    @Test
+    void deleteQueue() {
+        // Arrange
+        String queueName = testResourceNamer.randomName("test", 7);
+        QueueDescription description = new QueueDescription().setMaxDeliveryCount(15);
+        CreateQueueBody createEntity = new CreateQueueBody();
+        CreateQueueBodyContent content = new CreateQueueBodyContent()
+            .setType("application/xml")
+            .setQueueDescription(description);
+        createEntity.setContent(content);
+
+        logger.info("Creating queue: {}", queueName);
+
+        // This is not part of the scenario. We'll ensure it is created.
+        Response<Object> response = queuesClient.putWithResponseAsync(queueName, createEntity, null, Context.NONE)
+            .block(timeout);
+        assertNotNull(response);
+
+        // Act & Assert
+        StepVerifier.create(queuesClient.deleteWithResponseAsync(queueName, Context.NONE))
+            .assertNext(deletedResponse -> assertEquals(200, deletedResponse.getStatusCode()))
+            .verifyComplete();
```

Comment:
https://netty.io/4.0/api/io/netty/handler/codec/http/HttpResponseStatus.html#OK

Reply:
Same comment as above. You can explicitly exclude dependencies and this would tie them to netty instead of another http client.